iree_tests/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.vmfb
+# TODO(scotttodd): compile into a build dir instead of the source dir
+config_*.txt
```

Comment:
If using pytest, this can be replaced with https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html
If ctest, could use the build directory

For local iteration I do like keeping all related files in one folder though (could have pytest/ctest copy into the build/temp folder)